sources/gerritDB/helpers.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+exports.injectExcludedProjectsIntoSql = function (excludedProjects, projectColumnName) {
+    var colName = projectColumnName || "dest_project_name";
+    var sql = "";
+
+    if (!excludedProjects || excludedProjects.length === 0) {
+        return " true "; // don't fail conditions
+    }
+
+    excludedProjects.forEach(function (project, i) {
+        if (i === 0) {
```

Comment:
I think that's a tradeoff because following your suggestion also means losing the flexibility.